bitbots_vision/docs/manual/yoeo_vision.rst

```diff
@@ -0,0 +1,100 @@
+YOEO-based Vision (May 2023)
+================================================
+
+Description
+-----------
+
+Since 2022, the Bit-Bots vision relies on a Convolutional Neural Network called `YOEO
+<https://github.com/bit-bots/YOEO>`__. As of May 2023, YOEO provides object detection for balls and robots and semantic
+segmentations for field lines, the field itself and the background. Furthermore, it is able to discriminate between teammates and opponents in case of the robot class. Goal posts are supported code-wise, but not actively used at the moment.
+
+Supported Neural Network Frameworks
+-------------
+YOEOs can be run with the following frameworks: PyTorch, Onnx, OpenVino and TVM.
+
+How to Add a New YOEO Variant to the Vision?
+-------------
+In general, all models are stored in the ``models`` subdirectory. Therein, each YOEO variant comes with a dedicated
+subdirectory containing a file ``model_config.yaml`` as well as subdirectories named after the supported frameworks. The latter contain the actual model files for each framework. There is no need to create framework subdirectories (and the
+corresponding model files) for frameworks you do not plan to use.
+
+A full directory structure for a YOEO variant with the name "my_model" would look like this:
+
+::
+
+ /models
+     /my_model
+         /openvino
+             yoeo.bin
+             yoeo.xml
+         /onnx
+             yoeo.onnx
+         /pytorch
+             yoeo.cfg
+             yoeo.pth
+         /tvm
+             yoeo.json
+             yoeo.params
+             yoeo.so
+         model_config.yaml
+As the name already implies, the ``model_config.yaml`` provides information about the YOEO configuration. As of May 2023,
```

Comment:
For my taste, this is too much May 2023. It's likely, this will be forgotten to update in all three places